workflows/templates/trigger-integration-template.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ftrigger-integration
+  annotations:
+    thoth-station.ninja/template-version: 0.7.0
+  labels:
+    app: thoth
+    component: trigger-integration
+spec:
+  templates:
+    - name: trigger-integration
+      inputs:
+        parameters:
+          - name: "THOTH_WORKFLOW_NAME"
+          - name: "THOTH_DOCUMENT_ID"
+          - name: "THOTH_ADVISER_METADATA"
+      container:
+        name: trigger-integration
+        image: workflow-helpers
+        env:
+          - name: THOTH_WORKFLOW_TASK
+            value: "trigger_integration"
+          - name: APP_SCRIPT
+            value: "app.sh"
+          - name: THOTH_WORKFLOW_NAME
+            value: "{{inputs.parameters.WORKFLOW_NAME}}"
+          - name: THOTH_DOCUMENT_ID
+            value: "{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          - name: THOTH_ADVISER_METADATA
+            value: "{{inputs.parameters.THOTH_ADVISER_METADATA}}"
+          - name: "THOTH_WEBHOOK_CALLBACK_URL"
+            value: "http://qeb-hwt-aicoe-prod-bots.cloud.paas.psi.redhat.com"
+          - name: THOTH_WEBHOOK_SECRET
+            valueFrom:
+              secretKeyRef:
+                key: WEBHOOK_SECRET
+                name: qeb-hwt-github-app
+        resources:
+          limits:
+            cpu: 250m
+            memory: 256Mi
+          requests:
+            cpu: 250m
+            memory: 256Mi
```

Comment:
We need to add the outputs here

Reply:
thanks

Reply:
Added in 6a49c7663ec62f4f32da817857ad5354d10af343, Thanks @KPostOffice